webpack.config.js

```diff
@@ -215,7 +215,7 @@ if (NODE_ENV === "hot") {
       // NOTE Atte Keinänen 10/19/17: We are currently sticking to an old version of react-hot-loader
       // because newer versions would require us to upgrade to react-router v4 and possibly deal with
       // asynchronous route issues as well. See https://github.com/gaearon/react-hot-loader/issues/249
-      { loader: "react-hot-loader" },
+      { loader: "react-hot-loader/webpack" },
```

Comment:
This line was taken directly from @kdoh 's PR:
https://github.com/metabase/metabase/pull/13488/files#diff-1fb26bc12ac780c7ad7325730ed09fc4c2c3d757c276c3dacc44bfe20faf166fR211